Sources/InstantMock/MockUsable/Float+MockUsable.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Float+MockUsable.swift
+//  InstantMock
+//
+//  Created by Patrick on 06/05/2017.
```

Comment:
Hey @abarisain, you can put your name in the header file 😉

Reply:
Done! I'm leaving your copyright line though, to be consistent with the project. Thanks :)